src/components/binding/binding-loader.js

```diff
@@ -37,13 +37,32 @@ genotet.BindingLoader.prototype.LOCUS_MARGIN_RATIO = .1;
 genotet.BindingLoader.prototype.load = function(fileName, bedName, chr,
                                                 opt_track) {
   var trackIndex = opt_track ? opt_track : this.data.tracks.length;
+  var isAddTrack = !opt_track;
   this.data.chr = chr;
-  this.loadFullTrack(trackIndex, fileName, chr);
+  this.loadFullTrack(trackIndex, fileName, chr, isAddTrack);
   this.loadBed(bedName, chr, this.data.detailXMin, this.data.detailXMax);
   this.loadExons_(chr);
 };
 
 /**
+ * Loads the binding data for a given gene and chromosome in preset mode.
```

Comment:
Please also add comments to mention that the genes are added at the end of the tracks list.